packages/hash/shared/src/types.ts

```diff
@@ -1,393 +1,14 @@
-import { BaseUri, VersionedUri } from "@blockprotocol/type-system";
-import slugify from "slugify";
-import { systemUserShortname } from "@hashintel/hash-shared/environment";
-import { frontendUrl } from "./environment";
+// The types exported here are expected to be branded.
+// Perhaps some of these should be put into the subgraph stdlib instead?
 
-type SchemaKind = "data-type" | "property-type" | "entity-type";
+export type ownedById = string;
+export type AccountId = string;
 
-/** Slugify the title of a type */
-export const slugifyTypeTitle = (title: string): string =>
-  slugify(title, { lower: true });
+export type EntityId = string;
```

Comment:
Not sure I love this, we've already defined it elsewhere, I'd like to see an explicit task to migrate `EntityId` into this file, and _not_ include it here in the interim, or to just do that now and fix the imports.

Reply:
The only other place I've seen this is in the `packages/hash/subgraph/src/types/identifier.ts` file. I can remove it from here for now

Reply:
<img width="498" alt="image" src="https://user-images.githubusercontent.com/25749103/207118525-9074c5e6-6e22-4037-b306-532087c2c805.png">
Yeah I'm just saying `EntityId` is used a lot, let's not have competing definitions